maquetado/scss/titles.scss

```diff
@@ -1,6 +1,6 @@
 .title {
   font-size: $text-big;
-  margin: 50px 200px;
+  margin: 50px 200px 0;
```

Comment:
This margin is `margin: top left&right bottom`, are you sure you want this?

Reply:
At least answer me haha

Reply:
jajaja